cli/src/test/java/org/jboss/as/cli/parsing/test/CommandLineArgumentsTestCase.java

```diff
@@ -38,22 +37,6 @@
  */
 public class CommandLineArgumentsTestCase {
 
-    @Test
-    public void testDefault() throws Exception {
-
-        ParsedCommandLine args = parse("deploy ../../../../testsuite/smoke/target/deployments/test-deployment.sar --name=my.sar --disabled --runtime-name=myrt.sar --force");
-        assertTrue(args.hasProperties());
-        assertTrue(args.hasProperty("--name"));
-        assertTrue(args.hasProperty("--runtime-name"));
-        assertTrue(args.hasProperty("--disabled"));
-        assertTrue(args.hasProperty("--force"));
-
-        List<String> otherArgs = args.getOtherProperties();
-        assertEquals(1, otherArgs.size());
-        assertEquals("../../../../testsuite/smoke/target/deployments/test-deployment.sar", otherArgs.get(0));
-
-        assertNull(args.getOutputTarget());
-    }
```

Comment:
Does this no longer pass?

Reply:
I added it back. The operator passing is no more done by this parser but this class should still work.